docs/blog/newsletter-september-2023.rst

```diff
@@ -0,0 +1,111 @@
+
+When it comes to the actual writing, it helps to make the documentation process as efficient and repeatable as possible. Develop patterns for different types of documentation to make the task less burdensome for people who don't write the docs every day. Publish instructions to help people understand how to make small changes themselves and ask for help for big issues. Consider keeping a shorter, more streamlined style guide for internal docs. You might also be able to integrate docs work into other teams' workflows. For example, you could automatically create a docs ticket for every feature initiative opened by a product manager or require information about any relevant docs needs in bug report forms.
+
+Look for opportunities to encourage a sense of ownership among those who contribute to internal docs. When you act on someone's feedback or publish someone's contribution, make sure to follow up with a link to the result so that people know their efforts matter. Interested contributors might benefit from polishing their skills with `Google’s tech writing courses for software engineers <https://developers.google.com/tech-writing>`_.
+
+Other recommendations for further information:
+
+- `Any friend of the docs is a friend of mine: Cultivating a community of documentation advocates </videos/portland/2019/any-friend-of-the-docs-is-a-friend-of-mine-cultivating-a-community-of-documentation-advocates-heather-stenson/>`__ from Write the Docs Portland, 2019
+- `2022 State of DevOps Report data deep dive: Documentation is like sunshine <https://cloud.google.com/blog/products/devops-sre/deep-dive-into-2022-state-of-devops-report-on-documentation>`__
+- `Why it’s worth it to invest in internal docs <https://increment.com/documentation/why-investing-in-internal-docs-is-worth-it/>`__
+
+--------------------------
+Numbering Section Headings
+--------------------------
+
+To number or not was a question posed in recent months. In general, these queries related to printed (or PDF) manuals.
+
+Often reasons for numbering sections relate to using the number instead of the section title as a reference or to clarify about sequencing (or relationships) between sections. As one documentarian noted, "Titles alone do not tell users ... where they will find the information unless they are already ... familiar with the content structure of the document. When you have to check section 3, you know that it is before section 4 and ... after section 2."
```

Comment:
The first sentence seems a little hard to parse. Does it mean something like this?

`Often, the reasons for numbering section headings include using the number as a reference or clarifying the sequence of sections and relationships between sections.`

Reply:
```suggestion
Often, the reasons for numbering section headings relate to using the number as a reference or clarifying the sequencing of sections and relationships between sections. As one documentarian noted, "Titles alone do not tell users ... where they will find the information unless they are already ... familiar with the content structure of the document. When you have to check section 3, you know that it is before section 4 and ... after section 2."
```